### ISSUE TITLE
can't login & HTTP/500 on /doLogin

### ISSUE THREAD

**Issue by smarlowe-cisco** [jenkins-infra/helpdesk#3385]
### Type(s) of account issue

Other (please describe in the summary below)

### Summary

I tried to login to the issue tracker, but it says my username and password are incorrect.  It's been a while, so maybe I forgot my password.  So, I followed the can't access your account link to accounts.jenkins.io.  I tried to login there with the same result.  Next, I completed the forgot password flow.  I received an email with a temporary password and tried to login to both accounts.jenkins.io and the issue tracker.  Neither work with the temporary password.

Then I noticed that on accounts.jenkins.io, when I submit the login form it sends a POST to /doLogin and the response is an HTTP/500 error response.  Not sure, but that seems fishy to me.  I would expect an HTTP/401 (or maybe HTTP/403) for an incorrect password, not an HTTP/500.  So maybe there's something wrong with the authentication backend?

Also seems fishy that the temporary password I received via email doesn't work, so there's also that.

Regardless of the issue, I appreciate your help in getting logged in.

### jenkins.io account username

not sure, maybe smarlowe?

### jenkins.io account email

smarlowe@cisco.com

### Which URL are you trying to login at?

https://issues.jenkins.io/login.jsp
and
https://accounts.jenkins.io/login

**Comment by MarkEWaite** [comment]
I've sent a verification email message to confirm ownership of the email account.  Once that is confirmed, I can reset the password and provide the new password

**Comment by smarlowe-cisco** [comment]
My issue has now been resolved, and I'm able to login again.

My first try this morning resulted in another HTTP/500 on
accounts.jenkins.io.  After playing around a bit, I realized that
sending my email address (in the username field) resulted in the
HTTP/500.  But, if I used my actual username, then I was able to
login successfully.

Similar issue on issues.jenkins.io.  Trying to login with email
address failed with HTTP/200 message indicating my username or
password was incorrect.  When I switched to my actual username,
the login succeeded.

So, a big part of this was likely user error.  Having a good
sleep over the weekend helped me realize the form was asking for
username, not email.  Still seems odd to me that this caused an
HTTP/500 on accounts.jenkins.io though.

Thanks for the help.
